xonsh/completer.py

```
@@ -449,14 +469,14 @@ def option_complete(self, prefix, cmd):
                 manpage = subprocess.Popen(["man", cmd],
                                            stdout=subprocess.PIPE,
                                            stderr=subprocess.DEVNULL)
-                # This is a trick to get rid of reverse line feeds
+                # This is a trick to get rid of reverse line feeds.
                 text = subprocess.check_output(["col", "-b"],
                                                stdin=manpage.stdout)
                 text = text.decode('utf-8')
                 scraped_text = ' '.join(SCRAPE_RE.findall(text))
                 matches = INNER_OPTIONS_RE.findall(scraped_text)
                 self._options[cmd] = matches
-            except:
+            except Exception:  # pylint: disable=broad-except
```

Thanks, a bare except should never be allowed.